packages/console/src/pages/Connectors/CreateForm/index.tsx

```diff
@@ -140,7 +156,15 @@ function CreateForm({ onClose, isOpen: isFormOpen, type }: Props) {
             title="general.next"
             type="primary"
             disabled={!activeFactoryId}
-            onClick={() => {
+            onClick={async () => {
+              if (activeFactoryId === ServiceConnector.Email) {
+                const created = await createEmailServiceConnector();
```

Comment:
Could you explain the reason that this is special?

Reply:
updated.